Resources/doc/filters.md

```diff
@@ -0,0 +1,98 @@
+# Filters
+
+The bundle provides a generic system to apply filters on collections. It ships with built-in Doctrine ORM support
+and can be extended to fit your specific needs.
+
+By default, all filters are disabled. They must be enabled manually.
+
+When a filter is enabled, it is automatically documented as a `hydra:search` property in collection returns. It also automatically
+appears in the NelmioApiDoc documentation if this bundle is installed and enabled.
+
+## Adding Doctrine ORM filters
+
+If Doctrine ORM support is enabled, adding filters is as easy as adding an entry in your `app/config/services.yml` file.
+It supports exact and partial matching strategies. If the partial strategy is specified, a SQL query with a `LIKE %text to search%`
+query will be automatically issued.
+
+To allow filtering the list of offers:
+
+```yaml
+services:
+    resource.offer.filter:
+        parent:    "api.doctrine.orm.filter"
+        arguments: [ {
+                        "id": "exact",    # Filters on the id property, allow both numeric values and IRIs
+                        "price": "exact", # Extracts all collection elements with the exact given price
+                        "name": "partial" # Elements with given text in their name
+                   } ]
```

Comment:
Maybe an example passing several properties to show that it is possible? It's possible it's just me since I've seen the case where it was not possible in the previous version^^